### PR TITLE
avm2: Stub Stage.fullScreenWidth and fullScreenHeight

### LINKED ISSUE #9474: cat ninja not launching
i am aware that actionscript 3 content is not currently supported but I would like for it to be fixed. the problem is that it does not get past the loading screen. i am asking for a fix because it is a very simple game. thank you for your time.

### PR DESCRIPTION
**Pull request by n0samu**
Fixes #9474.

### CODE REVIEW

File: core/src/avm2/globals/flash/display/stage.rs [hunk 1]
```diff
@@ -522,3 +522,23 @@ pub fn set_full_screen_source_rect<'gc>(
...
 }
+
+/// Stage.fullScreenHeight's setter
+pub fn get_full_screen_height<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    avm2_stub_getter!(activation, "flash.display.Stage", "fullScreenHeight");
+    Ok(0.into())
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/display/stage.rs:533]: I suggest adding reasonable values here (1024x768 or 1920x1080), as this may control scaling on some games.

**Review by Lord-McSweeney — APPROVED** [review]

File: Cargo.lock [hunk 2]
```diff
@@ -2774,9 +2774,9 @@ dependencies = [
 
 [[package]]
 name = "orbclient"
-version = "0.3.42"
+version = "0.3.43"
```
Comment (Lord-McSweeney) [inline Cargo.lock:2777]: Was this change intentional?

Comment (n0samu) [reply Cargo.lock]: No, that was an accident. Fixed.

File: core/src/avm2/globals/flash/display/stage.rs [hunk 3]
```diff
@@ -522,3 +522,23 @@ pub fn set_full_screen_source_rect<'gc>(
     avm2_stub_setter!(activation, "flash.display.Stage", "fullScreenSourceRect");
     Ok(Value::Undefined)
 }
+
+/// Stage.fullScreenHeight's setter
```
Comment (torokati44) [inline core/src/avm2/globals/flash/display/stage.rs:526]: nit: "getter" in comment - also for the other method

**Review by torokati44 — APPROVED** [review]

File: core/src/avm2/globals/flash/display/stage.rs (hunk 3 above)
Comment (n0samu) [reply core/src/avm2/globals/flash/display/stage.rs]: Not sure how I managed to miss that 😬

File: core/src/avm2/globals/flash/display/Stage.as [hunk 4]
```diff
@@ -130,9 +130,13 @@ package flash.display {
         public native function get frameRate():Number;
         public native function set frameRate(value:Number):void;
 
+        public native function get fullScreenHeight():uint;
```
Comment (adrian17) [inline core/src/avm2/globals/flash/display/Stage.as:133]: for the future, it's alright to implement the stub in pure AS too :)

**Review by adrian17 — APPROVED** [review]
